From e099a2654aa25f98a5f9e7a0e1f0820e8322c372 Mon Sep 17 00:00:00 2001
From: Jakub Hrozek <jhrozek@redhat.com>
Date: Tue, 16 Aug 2016 20:59:30 +0200
Subject: [PATCH 30/39] SECRETS: Make reading the config options more uniform
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

One of confdb_get_ calls in sec_get_config() used a variable referenced
from rctx, the other used a hardcoded string. Use one of them on both
places instead.

Reviewed-by: Pavel Březina <pbrezina@redhat.com>
(cherry picked from commit ff35d4ffe5eedcb484deb7ef1a04c02c19e634c9)
---
 src/responder/secrets/secsrv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/responder/secrets/secsrv.c b/src/responder/secrets/secsrv.c
index 6f8052bdf25fe903683d6b26d92b2a4c31743470..eb194a179ae5e3a48547fb00a038f31b8e0264cd 100644
--- a/src/responder/secrets/secsrv.c
+++ b/src/responder/secrets/secsrv.c
@@ -35,7 +35,7 @@ static int sec_get_config(struct sec_ctx *sctx)
     int ret;
 
     ret = confdb_get_int(sctx->rctx->cdb,
-                         CONFDB_SEC_CONF_ENTRY,
+                         sctx->rctx->confdb_service_path,
                          CONFDB_SERVICE_FD_LIMIT,
                          DEFAULT_SEC_FD_LIMIT,
                          &sctx->fd_limit);
-- 
2.9.3

